config/install/linkit.linkit_profile.site_content.yml

```diff
@@ -0,0 +1,20 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - node
+id: site_content
+label: 'Site Content'
+description: ''
+matchers:
+  e94b5256-49c7-4b3c-bd2f-67e6a5a86929:
+    uuid: e94b5256-49c7-4b3c-bd2f-67e6a5a86929
+    id: 'entity:node'
+    weight: 0
+    settings:
+      include_unpublished: true
+      metadata: 'Sites: [node:field_node_site] | Node: [node:nid] | [node:field_publication:entity:title] | <strong>[node:published_status]</strong>'
```

Comment:
Because this token ` [node:field_publication:entity:title] ` is often empty or not applicable for the majority of content, can we look for a way to avoid it showing as a ` |  | ` before published status. 

Maybe It would benefit from having a label, to identify what it is.

```
'Sites: [node:field_node_site] | Node: [node:nid] | Parent: [node:field_publication:entity:title] | <strong>[node:published_status]</strong>'
```

Another option might be to create a custom token for this publication parent that returns as "none" if it is empty, or the publication title if it is used.

Reply:
@christopher-hopper good pickup - for a quick fix ive added the Parent label 
However this will show | Parent: | for the content types that dont have a parent publication

The other alternative I was thinking is in a custom token have the | as part of the token if the content is there.

Reply:
I'm not convinced that product team wouldn't just prefer the empty field. Let me ask in the JIRA ticket.